README.md

```diff
@@ -1,2 +1,107 @@
-# Pantry
-A Ruby on Rails backend for an Alexa Skill. 
+# Pantry Keeper
+
+A custom Amazon Alexa skill powered by a custom Ruby on Rails application. Pantry Keeper provides a modern, state-of-the-art solution to keeping track of grocery items and how long they will last. With ongoing development, Pantry Keeper has the potential to more new and innovative services to its end-users in the realms of food safety, food storage, and cooking.
```

Comment:
Probably intended to say something other than "potential to more new" in that last sentence.